CannibalisticZombies/Assets/PlayerCharacterController.cs

```diff
@@ -0,0 +1,45 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerCharacterController : MonoBehaviour
+{
+    public float moveSpeed;
+
+    public Transform orientation;
+
+    float horizontalInput;
```

Comment:
Per the codestyle guidelines, we want to make sure we always explicitly state the access modifier. 
Make sure to add `private` before each method and variable